include/rviz_visual_tools/rviz_visual_tools.h

```diff
@@ -594,6 +594,17 @@ class RvizVisualTools
                    const std::string &ns = "Path");
 
   /**
+   * \brief Display a marker of a series of connected colored lines
+   * \param path - a series of points to connect with lines
+   * \param colors - a series of colors
+   * \param radius - the thickness of the line
+   * \param ns - namespace of marker
+   * \return true on success
+   * \note path and colors vectors must be the same size
+   */  bool publishPath(const std::vector<Eigen::Vector3d> &path, const std::vector<colors> &colors, const double radius = 0.01,
+                   const std::string &ns = "Path");
```

Comment:
if you would like to be an active maintainer of this package, i can add you to the package.xml, README.md, and then I can create PRs for you to review whenever i am making changes. i would just need you to commit to reviewing them within a few days (more or less)